app/__init__.py

```diff
@@ -292,6 +298,7 @@ def setup_babel(application):
     application.babel = Babel(application)
     application.babel.localeselector(get_locale)
     application.jinja_env.add_extension('jinja2.ext.i18n')
+    application.jinja_env.cache = {}
```

Comment:
What i mean is I don't think it works locally either. it doesn't persist the cache across requests. il going to remove this as the default is a cache of 400 objects. which should be ample, however it works